src/main/java/com/code_intelligence/jazzer/mutation/engine/SeededPseudoRandom.java

```diff
@@ -73,6 +73,16 @@ public <T> int otherIndex(T[] array, int currentIndex) {
     }
   }
 
+  @Override
+  public int otherIndex(int length, int currentIndex) {
+    int otherIndex = currentIndex + closedRange(1, length - 1);
```

Comment:
The other overload can just call this method.

Reply:
Done!